src/Config/CliBuilder.ts

```diff
@@ -44,6 +44,48 @@ export class CliBuilder {
         describe:
           'What level of logs to report. Any logs of a higher level than the setting are shown.'
       })
+      .option('log-file', {
+        requiresArg: true,
```

Comment:
should it actually be required field ?

Reply:
It's yargs - that log-file param has to be followed by an argument:
https://yargs.js.org/docs/#api-reference-requiresargkey

The param itself is not [required](https://yargs.js.org/docs/#demandCommand).